firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigContainer.java

```diff
@@ -37,9 +37,9 @@ public class ConfigContainer {
   static final String PERSONALIZATION_METADATA_KEY = "personalization_metadata_key";
   static final String TEMPLATE_VERSION_NUMBER_KEY = "template_version_number_key";
   static final String ROLLOUT_METADATA_KEY = "rollout_metadata_key";
-  static final String ROLLOUT_METADATA_AFFECTED_KEYS = "affectedParameterKeys";
-  static final String ROLLOUT_METADATA_ID = "rolloutId";
-  static final String ROLLOUT_METADATA_VARIANT_ID = "variantId";
+  public static final String ROLLOUT_METADATA_AFFECTED_KEYS = "affectedParameterKeys";
```

Comment:
Doesn't block this PR, but wondering if these constants could be snake case like the others

Reply:
No, they're the key values within RolloutMetadata. They might actually be able to be moved to `RemoteConfigConstants` in the future.